### ISSUE TITLE
Support plug 1.0.0

### ISSUE THREAD

**Issue by c-rack** [mneudert/plug_accesslog#9]
https://github.com/elixir-lang/plug has released 1.0.0, now I get:

```
Looking up alternatives for conflicting requirements on plug
  From plug_accesslog v0.8.0: ~> 0.14
  From mix.exs: >= 1.0.0
```

**Comment by mneudert** [comment]
Using the (just released) new version 0.9.0 from hex should fix your problem. Otherwise it should also work if you configure your plug dependency with `override: true` (if you haven't already known that flag). Not necessary this time but might prevent some headaches in the future...

**Comment by c-rack** [comment]
That was fast, thanks! :+1:
